Runtime/Settings/AutopilotState.cs

```diff
@@ -67,6 +67,12 @@ public bool IsRunning
             }
         }
 
+        /// <summary>
+        /// This state Is invalid (read-only).
+        /// True if Autopilot is running and in Edit Mode, it is required to reset.
+        /// </summary>
+        internal bool IsInvalidState => IsRunning && !EditorApplication.isPlayingOrWillChangePlaymode;
```

Comment:
I think this line should be guarded with #if UNITY_EDITOR.

Reply:
Thanks!
fixed in 41b611c786522d940f258dacaf9bd13b015c2dce